inst

```diff
@@ -230,6 +230,29 @@ nextcloud_ensure_extended_attributes () {
         --position "cn=custom attributes,cn=univention,$ldap_base" \
         --set name=nextcloud
 
+    ICON_USER="/usr/share/univention-management-console-frontend/js/dijit/themes/umc/icons/scalable/nextcloudUser.svg"
+    ICON_GROUP="/usr/share/univention-management-console-frontend/js/dijit/themes/umc/icons/scalable/nextcloudGroup.svg"
+    [ -e "$ICON_USER" ] && rm "$ICON_USER"
+    ln -s "apps-$(univention-app get nextcloud component_id --values-only)".svg "$ICON_USER"
+    [ -e "$ICON_GROUP" ] && rm "$ICON_GROUP"
+    ln -s "apps-$(univention-app get nextcloud component_id --values-only)".svg "$ICON_GROUP"
+
+    univention-directory-manager settings/extended_options create "$@" \
+        --position "cn=nextcloud,cn=custom attributes,cn=univention,$ldap_base" --set module="users/user" \
+        --set name="nextcloudUser" \
+        --set isApp=1 \
+        --set default="1" \
+        --set editable=1 \
+        --set shortDescription='Nextcloud' \
+        --set longDescription='Whether user may access Nextcloud' \
```

Comment:
Suggestion for the longDescription: "User can access Nextcloud"

Reply:
Okay, i can adapt this. I just took the original description from the nextcloudEnabled attribute.